code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -296,7 +296,8 @@
 				for(var/obj/item/W in L)
 					if(!L.dropItemToGround(W))
 						qdel(W)
-				L.dust()
+				L.spawn_dust()
```

Comment:
I'd prefer if you just marked the rune as already working instead of caring about unrelated systems internals